play-ebean/src/main/java/play/db/ebean/EbeanParsedConfig.java

```diff
@@ -17,14 +17,21 @@
  */
 public class EbeanParsedConfig {
 
+  private final Boolean ddlGenerate;
   private final String defaultDatasource;
   private final Map<String, List<String>> datasourceModels;
 
-  public EbeanParsedConfig(String defaultDatasource, Map<String, List<String>> datasourceModels) {
+  public EbeanParsedConfig(
+      Boolean ddlGenerate, String defaultDatasource, Map<String, List<String>> datasourceModels) {
+    this.ddlGenerate = ddlGenerate;
```

Comment:
This constructor is the same story I described before at the other constructor, please follow what I described there also, thanks!